tests/Ingredients/DisplayRecipesTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
these tests fail

Reply:
works for me

Reply:
resolved - methods edited and test methods updated. All pass.